cmd/api/src/database/datapipestatus.go

```diff
@@ -42,15 +46,12 @@ func (s *BloodhoundDB) SetDatapipeStatus(ctx context.Context, status model.Datap
 		updateSql += ";"
 		return s.db.WithContext(ctx).Exec(updateSql, status, now).Error
 	}
-
 }
 
 func (s *BloodhoundDB) GetDatapipeStatus(ctx context.Context) (model.DatapipeStatusWrapper, error) {
 	var datapipeStatus model.DatapipeStatusWrapper
 
-	if tx := s.db.WithContext(ctx).Raw("SELECT status, updated_at, last_complete_analysis_at, last_analysis_run_at FROM datapipe_status LIMIT 1;").Scan(&datapipeStatus); tx.RowsAffected == 0 {
-		return datapipeStatus, sql.ErrNoRows
-	}
+	tx := s.db.WithContext(ctx).Raw("SELECT status, updated_at, last_complete_analysis_at, last_analysis_run_at FROM datapipe_status").First(&datapipeStatus)
```

Comment:
Should we make this a `.Select()` instead of `.Raw()` so it is consistent with `GetAnalysisRequest`?

https://github.com/SpecterOps/BloodHound/pull/976/files#diff-ea5f22a2117794302d1889f9392b6025c81b05a9d6d068f06676561cff6473b8R45

Reply:
Oh yes, good catch, will bring that up to date